bundle/edu.gemini.itc/src/test/java/edu/gemini/itc/gmos/CCDGapCalcTest.java

```diff
@@ -0,0 +1,24 @@
+package edu.gemini.itc.gmos;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Testing out calcIfu2Shift code in CCDGapCalc.
+ * [Add more details!!!]
+ * ...
+ */
+public class CCDGapCalcTest {
+
+    @Test
+    public void thisIsATest() {
+        final double actualResult0  = CCDGapCalc.calcIfu2Shift(750.0, 831.0);
+        final double expectedResult0 = 2736.24773782;           // Taken from gfoneshift in PYRAF
+        final double actualResult1  = CCDGapCalc.calcIfu2Shift(750.0, 150.0);
+        final double expectedResult1 = 3645.34948931;           // Taken from gfoneshift in PYRAF
+
+        Assert.assertTrue("this is true", true);
```

Comment:
Hi,

I don't know what line you are referring to, but I assume it is the one that says "[Add more details!!!]".

I've tried to add more details, but I'm having trouble doing a pull request. The line above should be replaced with the following text:

CCDGapCalc.java is based on the IRAF script gfoneshift.cl, which takes the central wavelength
and the lpmm and outputs the shift. This is used in the GMOS IFU-2 calculations.
This test was created to compare the output of CCDGapCalc.java to gfoneshift.cl.